.travis.yml

```diff
@@ -14,19 +14,48 @@
 
 dist: trusty
 sudo: required
-lang: cpp
+language: cpp
 
-compiler:
-  - clang
-  - gcc
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+      env: DISTRO=ubuntu DISTRO_VERSION=17.04
+    - os: linux
+      compiler: clang
+      env: DISTRO=ubuntu DISTRO_VERSION=16.04
+    - os: linux
+      compiler: gcc
+      env: DISTRO=ubuntu DISTRO_VERSION=14.04
+    - os: linux
+      compiler: gcc
+      env: DISTRO=fedora DISTRO_VERSION=27
+    - os: osx
+      compiler: clang
+
+script:
+  - ci/build-linux.sh
+  - ci/build-macosx.sh
+
+# Cache the (saved) Docker images.  With recent version of Docker one can
```

Comment:
`s/of Docker one can/of Docker, one can/`

Reply:
Done.